### PR TITLE
Update numi - version / url

### PR DESCRIPTION
**Pull request by commitay**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Simplify `version` and `url`.

### CODE REVIEW

**Comment by reitermarkus** [comment]
Doesn't the versioned URL work anymore?

**Comment by commitay** [comment]
The versioned `url` still works, this is easier to update.

**Comment by reitermarkus** [comment]
Nevertheless, versioned URLs are preferred since they don't break the `sha256` when a new version is released.

File: Casks/numi.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 cask 'numi' do
-  version '3.15.2,122:1496059169'
+  version '3.15.2'
   sha256 '16d6c2c0b8879d44217f0154f7cf5152b286c4c4a1049f9dbee76946719407b2'
 
   # dl.devmate.com/com.dmitrynikolaev.numi was verified as official when first introduced to the cask
-  url "https://dl.devmate.com/com.dmitrynikolaev.numi/#{version.after_comma.before_colon}/#{version.after_colon}/Numi-#{version.after_comma.before_colon}.zip"
+  url 'http://dl.devmate.com/com.dmitrynikolaev.numi/Numi.zip'
```
Comment (commitay) [inline Casks/numi.rb]: I didn't notice that this was `http`. Closing.

**Pull request closed without merging**